### PR TITLE
Add Mainnet/Ropsten Chainstack endpoints

### PR DESCRIPTION
**Pull request by easeev**
- Add Basic Auth support for commonAPI
- Add Mainnet/Ropsten Chainstack endpoints
- Trim trailing whitespaces

### CODE REVIEW

**Review by Hbutlercapone — COMMENTED** [review]
Added Mainnet/Ropsten chain stack endpoints

**Review by Hbutlercapone — COMMENTED** [review]
Added basic Auth Support for common API

**Review by Hbutlercapone — COMMENTED** [review]
word

**Review by Hbutlercapone — COMMENTED** [review]
word

**Review by Hbutlercapone — COMMENTED** [review]
- [ ]

**Review by antliv — COMMENTED** [review]
H

File: js/api.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
 function commonAPI(url, action, param){
```
Comment (mahammed1982) [inline js/api.js:1]: 0x5Bb3dbb6408C011e82DED83b236e1943c77c6600

**Review by mahammed1982 — COMMENTED** [review]
[https://mathwallet.org/secretnetwork-wallet/en/]()

File: js/api.js (hunk 1 above)
Comment (Hbutlercapone) [reply js/api.js:1]: Will it connect the api endpoints to this from multiple sites automatically

**Review by JRE1014 — COMMENTED** [review]
https://github.com/etherscan/ethvalidate/pull/12#pullrequestreview-805653468

**Review by Hbutlercapone — COMMENTED** [review]
Added mainnet/ropsten Chainstack endpoints #12

**Review by Hbutlercapone — COMMENTED** [review]
add #12

**Review by JRE1014 — COMMENTED** [review]
``gh pr checkout 12

**Review by JRE1014 — COMMENTED** [review]
https://www.paypal.com/Brudocs/outlook/validate-an-identity-token.md_C5DI7Fg4WrZPwOhII1dB0HEZ_qpNaYXEiy-o94UJ94zCr07GgrqMsfYQqFR7kn-mn68AjvLcgwSfZvyR_yIK75S_K37vC3QryQ7cNoafDe9upql_6pB2ybMVlgWPs_DmbJ8g0om-sPlwyn

D3HcPHqBMOKDPx4su1yKRLNErVD8xkx``

**Review by JRE1014 — COMMENTED** [review]
https://www.paypal.com/BruInstall-Package System.IdentityModel.Tokens.Jwt -Version 6.16.0

**Review by JRE1014 — COMMENTED** [review]
[https://github.com/etherscan/ethvalidate/pull/12#pullrequestreview-805653468](url)

[B3](https://www.paypal.com/Bru)

**Review by Cryptominer12 — COMMENTED** [review]
Duplicate of #`

**Review by Newman1515 — APPROVED** [review]

**Review by Pero110290 — APPROVED** [review]

**Review by F153 — COMMENTED** [review]
grazie mille

**Review by drvsinan — COMMENTED** [review]
Gr

**Comment by Vovanchik25071982** [comment]
> Added basic Auth Support for common API

**Review by Tannerrea — APPROVED** [review]

**Review by tankshark678 — COMMENTED** [review]
811c17bcfbfb371b4cab081d81893652914f9df4

**Pull request merged**